CHANGES.md

```diff
@@ -1,6 +1,427 @@
 Release Changes List
 ====================
 
+o spellcheck
+o quotes check
+o URL docs check (1.26)
+o linebreak check
+o check compiler flags / man pages
+o check example codes
+
+version 1.26.0
+==============
+
+Twenty-ninth public release of Chapel, March 31, 2022
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+* updated the Chapel compiler to support LLVM versions 11, 12, and 13
+* added a prototype ability to resize arrays of elements without default values
+* made stabilizing improvements to methods on ranges, domains, and arrays
+* added new 'CopyAggregation', 'Socket', and 'Channel' package modules
+* made stabilizing improvements to most standard library module interfaces
+* improved the portability, robustness, and performance of 'ofi' communication
+* expanded the generality of Chapel's ability to generate code for GPUs
+* added a new script for displaying available runtime configurations
```

Comment:
For the above three is it possible to give a little more specifics?  Like the name of the new script, function calls in kernels for the GPUs, and something like "especially SS11" for the ofi stuff?